src/actions/index.js

```diff
@@ -0,0 +1,287 @@
+export const CREATE_AUTH = 'CREATE_AUTH';
+export const LOGIN_AUTH_SUCCESS = 'LOGIN_AUTH_SUCCESS';
+export const LOGIN_AUTH_ERROR = 'LOGIN_AUTH_ERROR';
+export const GET_TOKEN_SUCCESS = 'GET_TOKEN_SUCCESS';
+export const GET_TOKEN_ERROR = 'GET_TOKEN_ERROR';
+export const RENEW_ID_TOKEN_SUCCESS = 'RENEW_ID_TOKEN_SUCCESS';
+export const RENEW_ID_TOKEN_ERROR = 'RENEW_ID_TOKEN_ERROR';
+export const DECODE_ID_TOKEN_SUCCESS = 'DECODE_ID_TOKEN_SUCCESS';
+export const DECODE_ID_TOKEN_ERROR = 'DECODE_ID_TOKEN_ERROR';
+export const REFRESH_SESSION_SUCCESS = 'REFRESH_SESSION_SUCCESS';
+export const REFRESH_SESSION_ERROR = 'REFRESH_SESSION_ERROR';
+export const CLOSE_SESSION_SUCCESS = 'CLOSE_SESSION_SUCCESS';
+export const CLOSE_SESSION_ERROR = 'CLOSE_SESSION_ERROR';
+export const LOGOUT_AUTH_SUCCESS = 'LOGOUT_AUTH_SUCCESS';
+export const LOGOUT_AUTH_ERROR = 'LOGOUT_AUTH_ERROR';
+
+const BASE_URL = "https://dev-570863.oktapreview.com";
+const CLIENT_ID = "OaCz3jBQxbaEnsDAFO3A";
+const REDIRECT_URI = "http://localhost:8080/";
+
+const RESPONSE_TYPE = ['id_token', 'token'];
+var CLIENT_SCOPES = ['openid', 'email', 'profile', 'groups'];
+
+
+// CREATE AUTH ACTIONS
+
+/**
+ *  Creates the Okta Authentication binding
+ */
+export function createAuth(uri) {
+  const request = new OktaAuth({
```

Comment:
Instead of request, I think I'd call this "authClient" since we're creating an instance of OktaAuth.